src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './App.css';
+import Calculator from './components/Calculator';
+
+function App() {
+  return (
+    <div>
+      <Calculator />
+    </div>
```

Comment:
- [ Required ] I think it would be wise if you increase the size of the `Calculator` component to make it easier to interact with, currently, the component appears to be too tiny to interact with. Please see the screenshot below for more details and improve on this.

## Your design
> The size of the Calculator is too small for a user who might be visually impaired, kindly make it a little bit bigger.

![Screenshot from 2023-08-17 02-53-02](https://github.com/fdezcaminero/math-magicians/assets/16392046/37f12185-bf5b-4f01-ba6b-444a33ce8d22)

## Project design
![](https://github.com/microverseinc/curriculum-react-redux/raw/main/math-magicians/images/calculator.png)